pkgconfig/pkgconfig.py

```diff
@@ -224,7 +224,8 @@ def split(m):
 
     result['define_macros'] = [split(m) for m in result['define_macros']]
 
-    return result
+    # only have members with values not being the empty list (which is default anyway):
+    return collections.defaultdict(list, [(k, v) for k, v in result.items() if v != []])
```

Comment:
Can you make the second parameter a generator to avoid creating yet another temporary list?

Reply:
can try... (intention was to keep it stupid-simple to make sure every python version understands it).